source/components/atoms/Select/Select.tsx

```diff
@@ -25,40 +26,57 @@ const pickerSelectStyles = StyleSheet.create({
 const Wrapper = styled.View`
   margin-bottom: 30px;
 `;
-
+const StyledErrorText = styled(Text)`
+  font-size: ${({ theme }) => theme.fontSizes[3]}px;
+  color: ${(props) => props.theme.textInput.errorTextColor};
+  font-weight: ${({ theme }) => theme.fontWeights[0]};
+  padding-top: 8px;
+`;
 interface Props {
   items: { label: string; value: string }[];
   onValueChange: (value: string, index?: number) => void;
   placeholder?: string;
   value: string;
   editable?: boolean;
   style?: ViewStyle;
+  onBlur: (value: string) => void;
+  showErrorMessage?: boolean;
+  error?: { isValid: boolean; message: string };
 }
 
 const Select: React.FC<Props> = React.forwardRef(({
   items,
   onValueChange,
+  onBlur,
   placeholder,
   value,
   editable = true,
+  showErrorMessage = true,
+  error,
   style,
 }, ref) => {
   const currentItem = items.find(item => item.value === value);
+  const handleValueChange = (itemValue: any) => {
```

Comment:
Should we not define the type of the itemValue? Or are there any underlaying reasons for the use of "any"?

Reply:
The any here comes from the picker-select library, it's from how it types its onValueChange prop. But of course it should be typed, I'll fix it.